cmd/joy/secret.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"github.com/nestoca/joy/internal/secret"
+	"github.com/spf13/cobra"
+)
+
+func NewSecretCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "secret",
+		Aliases: []string{"secrets", "sec"},
```

Comment:
I wonder if we could have a better name for this that makes it more obvious we're managing sealed-secrets and not just arbitrary secrets. Yes it says it in the description, but who reads the manual these days xD 

Maybe `sealed-secrets` with aliases `ss` and `ssecrets` ?

Reply:
Yeah I'm on the fence for this one, I get your point and I partly agree.  I still love simple and straight to the point command names, so I would propose we make `sealed-secrets` the official name of the command, that people will see when they do `joy --help`, however still provide `secret`, `secrets` and `sec` aliases.